fastlane/Fastfile

```diff
@@ -217,10 +217,10 @@ platform :ios do
   # the release_notes.txt file and the other text sources
   # -----------------------------------------------------------------------------------
   # Usage:
-  # bundle exec fastlane update_appstore_strings version:<release note version>
+  # bundle exec fastlane update_appstore_strings
```

Comment:
Technically the option is still available, just optional now, so I'd document it like this maybe?

```suggestion
  # bundle exec fastlane update_appstore_strings [version:…]
```

Alternatively we could use the occasion to convert the documentation of this lane to YARD docs like we now do in other repos (converting the doc of all the lanes in that repo is probably out of scope of this PR, but converting just this one could be a good start even if we don't yet update the others)